examples/3d/spotlight.rs

```diff
@@ -48,6 +48,8 @@ fn setup(
     ));
 
     // cubes
+
+    // Make it deterministic for testing purposes.
```

Comment:
```suggestion
    // We're seeding the PRNG here to make this example deterministic for testing purposes.
    // This isn't strictly required in practical use unless you need your app to be deterministic.
```

This may need to be reflected on the other examples.

Reply:
Do you want this exact working? I feel its a little bit verbose.

Reply:
It is but explaining the full reasoning is probably for the best.